apps/tutorial-servo-can-control/src/TutorialServo.cpp

```diff
@@ -0,0 +1,27 @@
+#include "TutorialServo.h"
+#include <algorithm>
+
+TutorialServo::TutorialServo(PinName servoPin, float servoRangeInDegrees = 180.0,
+						float minPulsewidthInMs = 1, float maxPulsewidthInMs = 2)

```

Comment:
Remove the default parameter values from the constructor definition. They should just be stated in the constructor declaration in the header file